src/main/java/com/three_tech_solutions/slot_app/data/mappers/MapperStudent.java

```diff
@@ -0,0 +1,24 @@
+package com.three_tech_solutions.slot_app.data.mappers;
+
+import com.three_tech_solutions.slot_app.controllers.responses.StudentResponse;
+import com.three_tech_solutions.slot_app.data.models.Student;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MapperStudent {
```

Comment:
El nombre correcto deberia ser `StudentMapper`, ya que deberia ser "Mapeador de estudiantes"

```suggestion
public class StudentMapper {
```